### ISSUE TITLE
[Suggestion]: Certain blocks should instantly be grabbed when the tractor beams go over it

### ISSUE THREAD

**Issue by SSgamingYT** [nonamecrackers2/crackers-wither-storm-mod#1977]
### Guidelines

- [X] By checking this box, I am saying that I have read the guidelines and believe to be following them to the best of my ability

### Description of feature request

Certain blocks such as string, tall grass, flowers, etc should be instantly grabbed by the tractor beams the moment they get touched by them

**Comment by Cod1ngGamer** [comment]
Makes sense, but should definitely be a config option because it might produce a lot of lag.

**Comment by SSgamingYT** [comment]
> Makes sense, but should definitely be a config option because it might produce a lot of lag.

True
